resources.py

```diff
@@ -14,6 +14,45 @@ def public_genomes_vds_path(split=False, version=CURRENT_RELEASE):
     return 'gs://gnomad-public/release/{0}/vds/genomes/gnomad.genomes.r{0}.sites{1}.vds'.format(version, ".split" if split else "")
 
 
+def get_gnomad_data(data_type, hardcalls=None, split=False, hail_version=CURRENT_HAIL_VERSION):
```

Comment:
Should we not have these be `get_gnomad_vds` and return a VDS ?
And then add options to keep release samples only, annotate metadata, etc.

Seems like this would add more functionality.

Reply:
Agreed, but then, thoughts on keeping this as get_gnomad_data_path() and then having another wrapper to get VDS with others? Sometimes you just want a path

Reply:
I feel like getting them directly from resources.py would be enough if just a path is needed. But open for discussion :)